### DISCUSSION TITLE
renovate/artifacts - error due to incorrect Java version

### DISCUSSION THREAD

**Discussion by igorwojda** [renovatebot/renovate#18343] [Request Help] [answered]
Hi

I am experiencing renovate check failure because the Gradle wrapper is executed using the check is performed on Java 8 instead of Java 11 (the Android Gradle plugin requires Java 11 to run).

http://github.com/igorwojda/android-showcase/pull/227#issuecomment-1268837089

How can I fix this?

**Comment by viceice** [comment]
are you using Gradle lockfiles?

which version of Gradle are you using?

**Comment by Churro** [comment, marked as answer]
Java constraints were mistakenly determined using the wrong base (current dep version instead of Gradle version). This bug is now addressed in PR https://github.com/renovatebot/renovate/pull/18346.
